Makefile

```diff
@@ -171,6 +171,14 @@ build-cli-cross-platform: ## Compiles the Lacework CLI for all supported platfor
             -ldflags=$(GO_LDFLAGS) \
             github.com/lacework/go-sdk/cli
 
+.PHONY: build-cli
+build-cli: ## Compiles the Lacework CLI for your platform
```

Comment:
what is the difference with `build` directive above? Should we replace it in favor of this one?

Reply:
I didn't see that.  Yeah, we should use `build` instead.